rest/serverless_test.go

```diff
@@ -52,7 +52,7 @@ func TestServerlessPollBuckets(t *testing.T) {
 	assert.Empty(t, configs)
 
 	// Create a database
-	rt2 := NewRestTester(t, &RestTesterConfig{CustomTestBucket: tb1, persistentConfig: true})
+	rt2 := NewRestTester(t, &RestTesterConfig{CustomTestBucket: tb1, persistentConfig: true, groupID: &sc.Config.Bootstrap.ConfigGroupID})
```

Comment:
How did this test work before?

Reply:
It was using the same test name and same bucket name so didn't need to worry about it needing the same group ID.